packages/react-scripts/package.json

```diff
@@ -53,12 +53,23 @@
     "jest": "18.0.0",
     "json-loader": "0.5.4",
     "object-assign": "4.1.0",
+    "postcss-cssnext": "^2.9.0",
+    "postcss-import": "^9.0.0",
     "postcss-loader": "1.0.0",
+    "postcss-modules-local-by-default": "^1.1.1",
     "promise": "7.1.1",
+    "react": "^15.3.0",
```

Comment:
I had to add this here because NPM was throwing peer dependency warnings. At the very end even with the warnings it would work because the script install the dependencies after. Not sure what to do with it, so I favoured a clear console.